src/ninjabrainbot/util/I18n.java

```diff
@@ -16,7 +16,7 @@
  * @since 2021/12/25 14:39
  */
 public class I18n {
-    private static final ResourceBundle BUNDLE = ResourceBundle.getBundle("resources/lang/I18n", Locale.getDefault(), new UTF8Control());
+    private static final ResourceBundle BUNDLE = ResourceBundle.getBundle("resources/lang/I18n", Locale.US, new UTF8Control());
```

Comment:
Also, how exactly does it know which language to choose, it seems to me that it will always be english based on this line.

Reply:
I forgot to change back to the original version when I was testing My English